apps/docs/content/guides/auth/auth-hooks/send-email-hook.mdx

```diff
@@ -6,6 +6,17 @@ subtitle: 'Use a custom email provider to send authentication messages'
 
 The Send Email Hook runs before an email is sent and allows for flexibility around email sending. You can use this hook to configure a back-up email provider or add internationalization to your emails.
 
+## Email Sending Behavior
+
+Email sending depends on two settings: Email Provider and Auth Hook status.
+
+| Email Provider Enabled | Auth Hook Enabled | Result                                                               |
```

Comment:
Maybe for better readability, switch the headers to

```suggestion
| Email Provider | Auth Hook | Result                                                               |
```

And then below use `Enabled` or `Disabled` depending on what the state is

Reply:
Good suggestion, let's do that. Thanks!